viewer/packages/cad-parsers/src/cad/V9GeometryFilterer.test.ts

```diff
@@ -0,0 +1,95 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+import { V9GeometryFilterer } from './V9GeometryFilterer';
+import { RevealGeometryCollectionType } from '@reveal/sector-parser';
+import * as THREE from 'three';
+
+import {
+  PrimitiveType,
+  Ellipsoid,
+  Box,
+  getCollectionType
+} from '../../../../test-utilities/src/primitives/primitiveTypes';
+
+import {
+  createPrimitiveInterleavedGeometry,
+  parseInterleavedGeometry
+} from '../../../../test-utilities/src/primitives/primitiveThreeBuffers';
+
+function assertSecondCulledAwayFromBoxAtX10Yneg10Z0(primitives: any[], primitiveType: PrimitiveType) {
+  const bufferGeometry = createPrimitiveInterleavedGeometry(primitiveType, primitives);
+
+  const clipBox = new THREE.Box3(new THREE.Vector3(8, -12, -2), new THREE.Vector3(12, -8, 2));
+
+  const collectionType = getCollectionType(primitiveType);
+
+  const filteredGeometry = V9GeometryFilterer.filterGeometry(bufferGeometry, collectionType, clipBox);
+
+  const newPrimitives = parseInterleavedGeometry(primitiveType, filteredGeometry);
+
+  expect(newPrimitives).toHaveLength(1);
+  expect(newPrimitives[0]).toContainEntries(Object.entries(primitives[0]));
+}
+
+describe('V9GeometryFilterer accepts empty clipBox', () => {
```

Comment:
We use `describe` as the 'category', and `test` to describe what's being tested. I think this test should group all tests together in one category.